GitUtils.ps1

```diff
@@ -159,13 +159,19 @@ function Get-GitStatus($gitDir = (Get-GitDirectory)) {
 
                 $indexAdded.AddRange($castStringSeq.Invoke($null, (,@($cacheResponse.IndexAdded))))
                 $indexModified.AddRange($castStringSeq.Invoke($null, (,@($cacheResponse.IndexModified))))
-                $indexModified.AddRange($castStringSeq.Invoke($null, (,@($cacheResponse.IndexRenamed | Select-Object -ExpandProperty Old))))
+                $indexRenamedOld = $cacheResponse.IndexRenamed.Old

```

Comment:
I just learned that you could do this today; what is this trick called (for search purposes)?

It returns either null (zero paths), a string (one path), or an object[] (many paths).